README.rst

```diff
@@ -256,9 +256,9 @@ directory of your choice::
 Synapse has a number of external dependencies, that are easiest
 to install using pip and a virtualenv::
 
-    virtualenv -p python3 env
-    source env/bin/activate
-    python -m pip install --no-use-pep517 -e ".[all]"
+    python3 -m venv ./env
+    source ./env/bin/activate
+    pip install -e ".[all,test]"
```

Comment:
```suggestion
    pip3 install -e ".[all,test]"
```

I think on systems that *still* have Python 2 as the default, `pip` may default to Python2 as well?

Reply:
Ah no wait, you've sourced the python environment, so it'll be Python 3!

Reply:
Activating the virtualenv in the preceding line should ensure that `pip` points to the version inside the venv.